astro/src/content/blog/announcing-fusionauth-1-49.mdx

```diff
@@ -0,0 +1,93 @@
+
+Let's look at some of the more notable fixes.
+
+## LinkedIn Provider Works Again
+
+In 2023, LinkedIn changed their implementation of ["Sign in with LinkedIn"](https://learn.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/sign-in-with-linkedin-v2). They made it more OIDC compliant (yay!). They also kept the old version around for older apps (yay!). But it broke FusionAuth's [LinkedIn Identity Provider](/docs/lifecycle/authenticate-users/identity-providers/social/linkedin) for new LinkedIn integrations (boo!).
+
+The new version is what all new "Sign in with LinkedIn" integrations should use. The older version is [deprecated as of Aug 2023](https://learn.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/sign-in-with-linkedin).
+
+In this release, FusionAuth uses the new LinkedIn Authentication API which is OIDC compatible. This includes more standard OIDC scopes, such as `openid` and `profile`, as well as a more standardized Id token which can be validated using [LinkedIn's discovery document](https://www.linkedin.com/oauth/.well-known/openid-configuration).
+
+We're happy LinkedIn updated their authentication workflow to support OIDC and are excited to support that integration going forward.
+
+## Elasticsearch Reindexing Progress
+
+I don't know about you, but I like progress indicators. I remember, years ago, watching progress indicators all the time, from network downloads to disk defragmentation programs. (RIP `Defrag`.) We all have faster internet and disk now, so hopefully those indicators are a thing of the distant past for you.
+
+But if you use FusionAuth and [reindex your Elasticsearch (or Opensearch) index](/docs/lifecycle/manage-users/search/search#reindexing-elasticsearch), before this release FusionAuth didn't do a great job of reporting progress. It only reported back every 250k objects, which depending on the data in the index, could give the impression that progress has halted.
+
+In this release and going forward, FusionAuth will report reindexing progress more regularly in order to spare everyone some frustration.
+
+## Email Link Workflow Improvements
+
+Speaking of frustration, FusionAuth has dealt with a lot of issues around [our magic links functionality](/docs/lifecycle/authenticate-users/passwordless/magic-links). This seems pretty straightforward:
+
+* create a customizable, localizable email template: check!
+* create a one-time use, random, high entropy string tied to a user account: check!
+* send the email to an account: check!
+
+If the user clicks on the link in the email, FusionAuth is assured they own the email address and should be logged in.  
+
+But, it's not quite that simple.
+
+Link checkers, which are common in corporate email settings, defeat this in the name of preventing phishing. They retrieve all the links in the email to make sure they are safe, invalidating the one-time code.
```

Comment:
I read this one like three times, I think this is what you are saying. Maybe this is more to the point? Or it is Tuesday and my 🧠 isn't working.

```suggestion
Corporate email link checkers prevent phishing attempts by automatically checking all links in an email. This means they access links before the user does, making one-time codes useless.
```